README.md

```diff
@@ -39,77 +40,79 @@ extraManifests:
+| `ingress.path`                              | Path to use in the `Ingress`. Should be set to `/*` if using gce-ingress in Google Cloud.                                                                                                                                                                                                                 | `/`                   |
+| `ingress.host`                              | Domain name Kubernetes Ingress rule looks for. Set it to the domain Atlantis will be hosted on.                                                                                                                                                                                                           |                       |
+| `ingress.hosts`                             | List of domain names Kubernetes Ingress rule looks for. Set it to the domains in which Atlantis will be hosted on.                                                                                                                                                                                        | `chart-example.local` |
+| `ingress.hosts[0].paths`                    | List of paths to use in Kubernetes Ingress rules.  Should be set to `/*` if using gce-ingress in Google                                                                                                                                                                                                   | `[/]`                 |
+| `ingress.tls`                               | Kubernetes tls block. See [Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/ingress/#tls) for details.                                                                                                                                                                            | `[]`                  |
+| `test.enabled`                              | Whether to enable the test.                                                                                                                                                                                                                                                                               | `true`                |
+| `extraManifests`                            | add additional manifests to deploy                                                                                                                                                                                                                                                                        | `[]`                  |
+| `initContainers`                            | Containers used to initialize context for Atlantis pods                                                                                                                                                                                                                                                   | `[]`                  |
```

Comment:
hey can you do the formatting in a separate PR?

Reply:
Sure, I reverted it from `main`.

Reply:
Sorry for the delay; I reverted original the formatting and added only the new field description in 56c4b8f